openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -370,7 +370,7 @@ def schedules_for_bin(self):
             })
             template_context.update(_get_upsell_information_for_schedule(user, schedule))
 
-            yield (user, schedule.enrollment.course.language, template_context)
+            yield (user, schedule.enrollment.course.closest_released_language, template_context)
```

Comment:
Same here, just want to be sure that this will be ok if `closest_released_language` is `None`.